addons/pandora/ui/components/entity_attributes/entity_attributes.gd

```diff
@@ -35,11 +39,18 @@ func init(entity:PandoraEntity) -> void:
 
 
 func _ready() -> void:
+	_translate()
 	texture_picker.texture_changed.connect(_set_icon_path)
 	script_picker.script_path_changed.connect(_set_script_path)
 	id_generation_enabled.toggled.connect(_set_id_generation)
 	class_name_edit.text_changed.connect(_set_class_name)
 	
+func _translate():
+	icon_label.text = TranslationManager.translate("ui.property_editor.icon_label.text")
```

Comment:
In Godot 3 we could simply use the `tr()` function available in GDScript. Could we do that instead of this custom translation mamager? Godot 4 supports .pot files in the project settings.

Reply:
I will take a better look at this, I thought that was only applied to the project, not to addons.

Reply:
Yeah, I ran some tests here and as I suspected the TranslationServer is not available in the EditorPlugin so we can't inject POT files.

Some refs:
https://github.com/godotengine/godot-proposals/issues/6885
https://github.com/godotengine/godot-proposals/issues/1262

I actually took the idea of the custom tr manager from the reference you gave in #65:

https://github.com/nathanhoad/godot_dialogue_manager/blob/71c65332cc7b34eee9c517aaa36d645a79bfadb3/addons/dialogue_manager/constants.gd#L254